src/Components/ArticleById.jsx

```diff
@@ -1,57 +1,68 @@
 import { useEffect, useState } from "react";
 import { getArticleById } from "../api";
-import { useParams } from "react-router-dom";
+import { Link, useParams } from "react-router-dom";
 import moment from "moment";
+import Loading from "./Loading";
+import CommentsCard from "./CommentsCard";
 
 function ArticleById() {
    const { articleId } = useParams();
    const [singleArticle, setSingleArticle] = useState([]);
+   const [isLoading, setIsLoading] = useState(false);
 
    useEffect(() => {
+      setIsLoading(true);
       getArticleById(articleId).then((article) => {
          setSingleArticle(article);
+         setIsLoading(false);
       });
    }, [articleId]);
 
+   if (isLoading) {
+      return <Loading />;
+   }
+
    if (!singleArticle) {
       return <h2>Article not found</h2>;
    }
 
    return (
-      <div>
-         <section className="single-article-container">
-            <h2 className="single-article-h2">{singleArticle.title}</h2>
-            <li className="articles-list" key={singleArticle.title}>
-               <p>
-                  <strong id="article-identifiers">
-                     Topic: {singleArticle.topic}
-                  </strong>
-               </p>
-               <p>
-                  <strong id="article-identifiers">Posted: </strong>
-                  {moment(singleArticle.created_at).format("DD/MM/YYYY")}
-               </p>
-               <p>
-                  <strong id="article-identifiers">Written by: </strong>
-                  {singleArticle.author}
-               </p>
-               <p>
-                  <strong id="article-identifiers">Votes: </strong>
-                  {singleArticle.votes}
-               </p>
-               <p>
-                  <strong id="article-identifiers">Comments: </strong>
-                  {singleArticle.comment_count}
-               </p>
-               <p>{singleArticle.body}</p>
-               <br></br>
-               <img
-                  className="article-imgs"
-                  src={singleArticle.article_img_url}
-               />
-            </li>
-         </section>
-      </div>
+      <section className="single-article-container">
+         <h2 className="single-article-h2">{singleArticle.title}</h2>
+         <br />
+         <li className="articles-list" key={singleArticle.title}>
+            <p className="single-article-identifiers">
+               {singleArticle.topic} | Written by {singleArticle.author} |{" "}
+               {moment(singleArticle.created_at).format("DD/MM/YYYY")}
+            </p>
+            <br />
```

Comment:
Try to avoid use line breaks for this, I'd consider using CSS instead for styling 👀